assist/README.md

```diff
@@ -1,8 +1,141 @@
 RACK ASSIST
 ===========
 
+# Introducing RACK ASSIST
+
 (Automated Software Structure Ingestion and System Testing)
 
+The ASSIST toolkit contains several tools that can be used with RACK.
+
+ * ASSIST-DC  -- Data Collection
+ * ASSIST-DI  -- Data Ingestion
+ * ASSIST-DV  -- Data Verification
+ * ASSIST-SADL  -- SADL translation
+ * ASSIST-RACKLIB -- RACK library
+
+## ASSIST-DC  -- Data Collection
```

Comment:
Could we maybe add a blockquote at the beginning of each section to indicate which concrete scripts relate to the abstract tool?

Something like:

> Where: `databin/{ar,gcc,make,tar,...}`

> Where: `bin/analyze`

> Where: `bin/sadl2owl`

Reply:
Good idea!